lib/validators/allowed_ip_address_validator.rb

```diff
@@ -3,12 +3,10 @@
 class AllowedIpAddressValidator < ActiveModel::EachValidator
   def validate_each(record, attribute, value)
     if record.ip_address
-      if ScreenedIpAddress.should_block?(record.ip_address)
-        record.errors.add(attribute, I18n.t("user.ip_address.blocked"))
-      end
+      record.errors.add(attribute, :blocked) if ScreenedIpAddress.should_block?(record.ip_address)
```

Comment:
This is possible now that we have moved the error message translations into a location that can be inferred by ActiveModel.